scene/gui/text_edit.cpp

```diff
@@ -1102,6 +1090,38 @@ void TextEdit::_notification(int p_what) {
 								}
 								draw_rect(rect, selection_color, true);
 							}
+
+							// End of line selected
+							if (line < get_selection_to_line(c)) {
```

Comment:
Would it worth pushing back into `Vector<Vector2> sel` rather then duplicating the validation code?

Reply:
Yes it would. Way neater.